<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTask</class>
 <widget class="QDialog" name="DialogTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="textEditSolution"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonOk">
     <property name="text">
      <string>Ответить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowserTask"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Решение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Текст задачи</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
